packages/client/src/components/form/DocumentUploadfield/SimpleDocumentUploader.tsx

```diff
@@ -125,16 +140,24 @@ class SimpleDocumentUploaderComponent extends React.Component<
   }
 
   render() {
-    const { label, intl, files, description, error, disableDeleteInPreview } =
-      this.props
-    const errorMessage = this.state.error || error || ''
+    const {
+      label,
+      intl,
+      files,
+      description,
+      error,
+      disableDeleteInPreview,
+      requiredErrorMessage,
+      touched
+    } = this.props
+    const errorMessage = requiredErrorMessage || this.state.error || error || ''
 
     return (
       <>
         {description && <FieldDescription>{description}</FieldDescription>}
         <ErrorMessage>
-          {errorMessage && (
-            <ErrorText id="field-error">{errorMessage}</ErrorText>
+          {errorMessage && touched && (
+            <ErrorText id="field-error">{errorMessage as string}</ErrorText>
```

Comment:
Have you tested what happens when you submit the `requiredErrorMessage` parameter as a MessageDescriptor object? Wouldn't that cause an object being rendered here as a string?

Reply:
Yeah, I tested. and worked fine. 

![Screenshot from 2022-03-02 12-30-59](https://user-images.githubusercontent.com/43314141/156307835-224c786a-4a88-4c9a-8ea4-ab2ae81b604d.png)

this is the message from requiredErrorMessage.